### ISSUE TITLE
Undefined variable $running

### ISSUE THREAD

**Issue by UteHaus** [mtierltd/timetracker#159]
[PHP] Fehler: Undefined variable $running at /var/www/html/custom_apps/timetracker/lib/Controller/AjaxController.php#262

POST /index.php/apps/timetracker/ajax/delete-work-interval/7

```
{"reqId":"FR42frTgNKjw7oHyIYxo","level":3,"time":"2022-01-16T10:07:43+00:00","remoteAddr":"93.202.133.174","user":"adi2","app":"PHP","method":"POST","url":"/index.php/apps/timetracker/ajax/delete-work-interval/7","message":"Undefined variable $running at /var/www/html/custom_apps/timetracker/lib/Controller/AjaxController.php#262","userAgent":"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:95.0) Gecko/20100101 Firefox/95.0","version":"23.0.0.10","exception":{"Exception":"Error","Message":"Undefined variable $running at /var/www/html/custom_apps/timetracker/lib/Controller/AjaxController.php#262","Code":0,"Trace":[{"file":"/var/www/html/custom_apps/timetracker/lib/Controller/AjaxController.php","line":262,"function":"onError","class":"OC\\Log\\ErrorHandler","type":"::","args":[2,"Undefined variable $running","/var/www/html/custom_apps/timetracker/lib/Controller/AjaxController.php",262]},{"file":"/var/www/html/lib/private/AppFramework/Http/Dispatcher.php","line":217,"function":"deleteWorkInterval","class":"OCA\\TimeTracker\\Controller\\AjaxController","type":"->","args":["7"]},{"file":"/var/www/html/lib/private/AppFramework/Http/Dispatcher.php","line":126,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[{"__class__":"OCA\\TimeTracker\\Controller\\AjaxController"},"deleteWorkInterval"]},{"file":"/var/www/html/lib/private/AppFramework/App.php","line":157,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[{"__class__":"OCA\\TimeTracker\\Controller\\AjaxController"},"deleteWorkInterval"]},{"file":"/var/www/html/lib/private/Route/Router.php","line":302,"function":"main","class":"OC\\AppFramework\\App","type":"::","args":["OCA\\TimeTracker\\Controller\\AjaxController","deleteWorkInterval",{"__class__":"OC\\AppFramework\\DependencyInjection\\DIContainer"},{"id":"7","_route":"timetracker.ajax.delete_work_interval"}]},{"file":"/var/www/html/lib/base.php","line":1006,"function":"match","class":"OC\\Route\\Router","type":"->","args":["/apps/timetracker/ajax/delete-work-interval/7"]},{"file":"/var/www/html/index.php","line":36,"function":"handleRequest","class":"OC","type":"::","args":[]}],"File":"/var/www/html/lib/private/Log/ErrorHandler.php","Line":92,"CustomMessage":"--"},"id":"61e3f0b76dbe9"}

```

Nextcloud: 23
Time Tracker:  0.0.73

**Comment by Weiming-Hu** [comment]
This change is probably needed as well for the function `addWorkInterval` because I'm getting the following error:

```
Error: Undefined variable $running at /var/www/html/nextcloud/apps/timetracker/lib/Controller/AjaxController.php#460
```

Nextcloud: 23.0.6
Time Tracker: 0.0.74

**Comment by mtierltd** [comment]
@Weiming-Hu should be fixed by a553cfeea1301a29152d86c1e793e4375f5f233c
